manubot/process/bibliography.py

```diff
@@ -63,9 +62,9 @@ def load_manual_references(paths=[], extra_csl_items=[]) -> dict:
         bibliography = load_bibliography(path)
         for csl_item in bibliography:
             csl_item.note_append_text(
-                f"This CSL JSON Item was loaded by Manubot v{manubot_version} from a manual reference file."
+                f"Loaded from an external bibliography file by Manubot."
             )
-            csl_item.note_append_dict({"manual_reference_filename": path_obj.name})
+            csl_item.note_append_dict({"bibliography": path_obj.name})
```

Comment:
bibliography is more consistent with the Pandoc vocabulary than `manual_reference_filename`.

Reply:
updated to `source_bibliography`